test/test_pipeline/components/training/base.py

```diff
@@ -42,7 +43,7 @@ def prepare_trainer(self,
 
         elif task_type in REGRESSION_TASKS:
             X, y = make_regression(
-                n_samples=5000,
+                n_samples=500,
```

Comment:
Why not usig here the `N_SAMPLES` global variable you created? (or something similar)

Reply:
that is completely true, I'll change to that.